build-system/tasks/compile.js

```diff
@@ -256,7 +256,7 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
       'node_modules/set-dom/src/**/*.js',
       'node_modules/web-animations-js/web-animations.install.js',
       'node_modules/web-activities/activity-ports.js',
-      'node_modules/@ampproject/worker-dom/dist/**/*.js',
+      'node_modules/@ampproject/worker-dom/dist/unminified.index.safe.mjs.js',
```

Comment:
If we're changing the extension below, lets fix it up here too.

```suggestion
      'node_modules/@ampproject/worker-dom/dist/unminified.index.safe.js',
```